Tests/StarknetTests/Data/TypedDataTests.swift

```diff
@@ -47,14 +47,14 @@ final class TypedDataTests: XCTestCase {
     static let exampleDomainV0 = """
     {
         "name": "DomainV0",
-        "version": 1,
+        "version": "1",
         "chainId": 2137,
     }
     """
     static let exampleDomainV1 = """
     {
         "name": "DomainV1",
-        "version": 2,
+        "version": "2",
```

Comment:
note: We wouldn't have to change this, but our tests compare original json and the one after decoding+encoding, hence it fails (version in original is int and in encoded it's string).